app/adapters/alma_adapter.rb

```diff
@@ -46,6 +46,95 @@ def get_availability_many(ids:)
+    # ...create the availability response for each item
+    availability = []
+    holding_items[:items].each do |item|
+      item_data = item["item_data"] || {}
+      holding_data = item["holding_data"] || {}
+
+      location = item_data["location"] || {}
+      library = item_data["library"] || {}
+      in_temp_library = false
+      temp_library = {}
+      temp_location = {}
+      if holding_data["in_temp_location"]
+        in_temp_library = true
+        temp_library = holding_data["temp_library"] || {}
+        temp_location = holding_data["temp_location"] || {}
+      end
+
+      status_label = marc_holding["availability"]
+      if status_label.nil?
+        # TODO: Figure out if this is right.
+        #     When the marc_holding is nil it seems that there is a a single holding in the bib but
+        #     that holding does NOT have an ID, yet, I think for eresources that single holding
+        #     seems to have the data that we need and therefore we could take the "availability"
+        #     property from that holding.
+        #     For an example see: http://localhost:3000/bibliographic/9919392043506421/holdings/22105104420006421/availability.json
+        # For now use the data in the item.
+        #
+        # If we do use the item_data value we should normalize it so that it says "available"
+        # rather than "Item in place".
+        status_label = item_data["base_status"]["desc"]
+      end
+
+      policy = item_data["policy"] || {}
+      item_av = {
+        barcode: item_data["barcode"],
+        id: item_data["pid"],
+        copy_number: holding_data["copy_id"],
+        status: nil,                                # ?? "Not Charged"
+        on_reserve: nil,                            # ??
+        item_type: policy["value"],                 # Gen
+        pickup_location_id: location["value"],      # stacks
+        pickup_location_code: location["value"],    # stacks
+        location: library["value"],                 # firestone
+        label: library["desc"],                     # Firestore Library
+        in_temp_library: in_temp_library,
+        status_label: status_label,                 # available
+        description: item_data["description"],      # "v. 537, no. 7618 (2016 Sept. 1)" - new in Alma
+        enum_display: item.enumeration,             # in Alma there are many enumerations
+        chron_display: item.chronology              # in Alma there are many chronologies
+      }
+
+      if in_temp_library
+        item_av[:temp_library_code] = temp_library["value"]
+        item_av[:temp_library_label] = temp_library["desc"]
+        item_av[:temp_location_code] = temp_library["value"]
+        item_av[:temp_location_label] = temp_library["desc"]
+      end
+
+      availability << item_av
+    end
+
+    # The `total_count` property might seem extra here but this is in preparation
+    # for when the client supports pagination and requests only a page of records.
+    response = {
```

Comment:
Question: is this what the response looks like in production, or will we have to change requests to support this?

Reply:
This will mean a change in requests. Hector brought this up in an alma checkin but we will also run this by @carolyncole

Reply:
Doesn't that mean we have to change the `main` branch?

Reply:
There are other changes and while this one we could avoid I don't think we could avoid all of them because of the hit rate on the alma API. I have been thinking we need to figure out our strategy here for requests, but I don't have a proposal yet.

Reply:
Maybe time to revisit an `alma` branch on OL?

Reply:
Yeah, I'm open to that. Or maybe an alma branch on requests and using an environment variable to specify the branch name in the gemfile.

Reply:
I changed back the response to be identical to what the Request app expects (i.e. an array rather than an object with an array inside.)